Twig/Extension/FormFlowExtension.php

```diff
@@ -36,8 +41,14 @@ public function getFilters() {
 	 * {@inheritDoc}
 	 */
 	public function getFunctions() {
+		if (version_compare(\Twig_Environment::VERSION, '1.12', '<')) {
```

Comment:
No need for a new major version here: Symfony 2.3 relies on Twig ``~1.23`` anyway. So the only way to use Twig 1.11 or older is to use unmaintained versions of Symfony. And updating Twig to the latest 1.x is a no-brainer thanks to BC (and Twig never provides bug fixes for older minor versions, so everyone is expected to update)